login/service_whitebox_test.go

```diff
@@ -188,8 +187,7 @@ func TestEncodeTokenOK(t *testing.T) {
 }
 
 func TestInt32ToInt64OK(t *testing.T) {
-	var i32 int32
-	i32 = 60
+	i32 := 60
```

Comment:
Thanks done in https://github.com/fabric8-services/fabric8-wit/pull/1489/commits/11a85853665905840341dc5a1df3545d84c0eaea#diff-ce0aa7f266a3f01ad173ef2e98624125R191